### DISCUSSION TITLE
Putting it on webserver completely stops it from working

### DISCUSSION THREAD

**Discussion by TheXer** [kuzmoyev/google-calendar-simple-api#81] [General]
Hello, one and possibly last issue I have with this is the error im encountering when I put it on hosting. 

OSError: [Errno 99] Cannot assign requested address

The problem is, I have correct keys from Google (as I have registered domain to work with) and put the correct values in this line: 
```
calendar = GoogleCalendar(calendar='email@gmail.com', authentication_flow_host= "164.132.110.138", authentication_flow_port=10038)
```
I found out about it in your documentation, so I tried everything possible. 
It shows me this error. How can I prevent it? How can I reconfigure the hosting etc.? 

Thanks for the help!

**Comment by kuzmoyev** [comment]
@TheXer Do you run it in docker? Does [this](https://medium.com/@simicode/oserror-errno-99-cannot-assign-requested-address-when-starting-dockerized-web-app-54c11f7041ea) help?

**Reply by TheXer** [reply]
Unfortunately doesn't, as I don't have access to that. I have bash, but just some of the commands are available. Is there another option? 

I tried to turn off the webserver and that didn't help at all.

**Reply by kuzmoyev** [reply]
Try using `authentication_flow_host="0.0.0.0"`. 

Apart from that, I don't think I can help you as it's a very specific problem and doesn't seem to be related to the `gcsa` but to the configuration of your server.

**Reply by TheXer** [reply]
Unfortunately doesn't work, i'll try to look at Google credentials if that's something that inteferes with it. Thanks for the help, though

**Reply by kuzmoyev** [reply]
Let me know if I can help with anything. And if you figure out the issue, please leave a comment about the cause. Might be useful for others or could be a potential fix that is needed.

Thanks!

**Reply by TheXer** [reply]
Unfortunately I found no solution so I settled to use raspberry pi as hosting to my discord bot so I can use local host and works fine since. 

Thanks anyway for this beginner friendly API wrapper and I wish you best of luck in future deployments.
